plugins-weekly.ci.jenkins.io.txt

```diff
@@ -26,7 +25,6 @@ coverage:1.7.0
 command-launcher:107.v773860566e2e
 commons-lang3-api:3.13.0-62.v7d18e55f51e2
 commons-text-api:1.11.0-95.v22a_d30ee5d36
-config-file-provider:959.vcff671a_4518b_
```

Comment:
Note that this plugin is still installed as it is a dependency of `job-dsl`:

```
[2024-01-09T18:47:08.571Z] #23 40.62 job-dsl depends on: 
[2024-01-09T18:47:08.571Z] #23 40.62 configuration-as-code 1646.v98b_72d209775
[2024-01-09T18:47:08.571Z] #23 40.62 branch-api 2.1105.v472604208c55
[2024-01-09T18:47:08.571Z] #23 40.62 cloudbees-folder 6.815.v0dd5a_cb_40e0e
[2024-01-09T18:47:08.571Z] #23 40.62 config-file-provider 938.ve2b_8a_591c596
```

Reply:
config-file-provider is an optional dependency only, https://github.com/jenkinsci/job-dsl-plugin/blob/fb7d49a2d5a002bf82449cdf96c4b98ea8a630ae/job-dsl-plugin/pom.xml#L103

Optional dependencies are installed when needed, but given that the plugin is unistallable from the web ui, job-dsl doesn't hard depend on it:
![Screenshot 2024-01-10 at 09 46 11](https://github.com/jenkins-infra/docker-jenkins-weekly/assets/13383509/60da5cee-4eb0-47ce-9e45-6015285467ba)